### DISCUSSION TITLE
Accessing environement variables through `process.env`?

### DISCUSSION THREAD

**Discussion by julienmru** [tighten/jigsaw#570] [Q&A]
Hello,

I am building an app where some environment variables will defer between local and production environments and I'd like to access those environment variables through JS.

Have you got any idea on how to do so? It seems `process.env` is not available in my JS.

I also tried to add the following in my webpack.mix.js file with no success:

```
mix.webpackConfig(webpack => {
    return {
        plugins: [
            new webpack.DefinePlugin({
			  PRODUCTION: JSON.stringify(mix.inProduction()),
            })
        ]
    };
});
```

Thanks in advance for your help :)

**Comment by bakerkretzmar** [comment]
Does `process.env.NODE_ENV` in your JavaScript files work? That should be a string like `development` or `production`, I'm not sure why but it looks like you can access individual environment variables directly but not the whole `process.env`.
